### PR TITLE
Add geo-related examples

### PR DESCRIPTION
**Pull request by judell**
# Integration test logs
<details>
  <summary>Logs</summary>
  
```
Add passing integration test logs here
```
</details>

# Example query results
<details>
  <summary>Results</summary>
  
```
Add example SQL query results here (please include the input queries as well)
```
</details>

### CODE REVIEW

File: docs/tables/twitter_search_recent.md [hunk 1]
```diff
@@ -62,3 +62,30 @@ from
...
+
+### Place, author, and text for tweets about the weather in Vermont
+
+```
+select 
+  place ->> 'full_name',
+  author ->> 'username' as author,  
+  text 
+from 
+  twitter.twitter_search_recent
```
Comment (cbruno10) [inline docs/tables/twitter_search_recent.md:74]:
```suggestion
  twitter_search_recent
```

Assuming we don't need the connection name, I think we can remove it?

File: docs/tables/twitter_search_recent.md [hunk 2]
```diff
@@ -62,3 +62,30 @@ from
...
+
+### Tweets about weather within 10 miles of a lat/lon location 
+
+Note: `point_radius` and related operators are not available with a basic ("Essential") account, see [operators by product](https://developer.twitter.com/en/docs/twitter-api/enterprise/rules-and-filtering/operators-by-product).
+
+```
+select 
+  *
+from 
+  twitter.twitter_search_recent
```
Comment (cbruno10) [inline docs/tables/twitter_search_recent.md:88]:
```suggestion
  twitter_search_recent
```

File: docs/tables/twitter_search_recent.md [hunk 3]
```diff
@@ -62,3 +62,30 @@ from
 where
   query = 'url:steampipe.io'
 ```
+
+### Place, author, and text for tweets about the weather in Vermont
+
+```
+select 
+  place ->> 'full_name',
```
Comment (cbruno10) [inline docs/tables/twitter_search_recent.md:70]:
```suggestion
  place ->> 'full_name' as place,
```

Should this column have an alias as well?

**Review by cbruno10 — CHANGES_REQUESTED** [review]
Please see comments, thanks!

**Comment by judell** [comment]
Yes to all three, thanks @cbruno10

**Review by cbruno10 — APPROVED** [review]

**Pull request merged**